NodeBase/entry_point.sh

```diff
@@ -29,19 +29,17 @@ if [ ! -z "$SE_OPTS" ]; then
   echo "appending selenium options: ${SE_OPTS}"
 fi
 
-# TODO: Look into http://www.seleniumhq.org/docs/05_selenium_rc.jsp#browser-side-logs
-
```

Comment:
Removing this TODO since  the `browserSideLog` (which the URL refers to), should not be used anymore according to the Selenium Grid [code](https://github.com/SeleniumHQ/selenium/blob/master/java/server/src/org/openqa/grid/internal/utils/configuration/StandaloneConfiguration.java#L101)